presto-docs/src/main/sphinx/functions/aggregate.rst

```diff
@@ -106,6 +107,7 @@ General Aggregate Functions
 .. function:: min(x) -> [same as input]
 
     Returns the minimum value of all input values.
+    When x is a REAL or DOUBLE returns NaN if all input values are NaN. Ignores NaN values otherwise.
```

Comment:
```suggestion
    When x is a REAL or DOUBLE, returns NaN if all input values are NaN. Ignores NaN values otherwise.
```
Suggest comma as a break for readability, to separate the condition from the outcome.